crates/edr_napi/src/account.rs

```diff
@@ -1,28 +1,134 @@
+use edr_solidity_tests::{backend::Predeploy, revm::state::AccountInfo};
 use napi::bindgen_prelude::{BigInt, Uint8Array};
 use napi_derive::napi;
 
+use crate::{
+    cast::TryCast,
+    serde::{
+        serialize_bigint_as_struct, serialize_optional_bigint_as_struct,
+        serialize_optional_uint8array_as_hex, serialize_uint8array_as_hex,
+    },
+};
+
 /// Specification of overrides for an account and its storage.
 #[napi(object)]
+#[derive(Clone, serde::Serialize)]
```

Comment:
Needs `Serialize` as the soltest config is `Serialize`

Reply:
See https://github.com/NomicFoundation/edr/pull/971